types/sinon/ts3.1/sinon-tests.ts

```diff
@@ -82,6 +82,7 @@ function testSandbox() {
     const stubInstance = sb.createStubInstance(cls);
     const privateFooStubbedInstance = sb.createStubInstance(PrivateFoo);
     stubInstance.foo.calledWith('foo', 1);
+    stubInstance.foo.calledWith('foo');
```

Comment:
Won't still allow stuff like `.calledWith('foo', undefined)` which would be invalid?

Reply:
a partial doesn't include extra values.

as in, `Partial<[number, number]>` would not match `[1, 3, 4]`. if your inferred arguments allow for an `undefined` in the second position, sure.